src/cascadia/TerminalApp/Profile.cpp

```diff
@@ -794,6 +809,77 @@ std::string_view Profile::_SerializeCloseOnExitMode(const CloseOnExitMode closeO
     }
 }
 
+int32_t Profile::ParseSnapOnOutput(const Json::Value& json)

```

Comment:
I need to add a doc.

Also, the implementation for this section is up for debate in the spec.